harness/determined/pytorch/deepspeed/_deepspeed_trial.py

```diff
@@ -16,10 +16,10 @@
 from deepspeed.runtime import dataloader as ds_loader
 
 import determined as det
-from determined import layers, pytorch, tensorboard, util, workload
+from determined import layers, pytorch, util, workload
 from determined.common import storage
 from determined.pytorch import deepspeed as det_ds
-from determined.tensorboard.metric_writers.pytorch import TorchWriter
+from determined.tensorboard.metric_writers.util import is_numerical_scalar
```

Comment:
google style guide for python imports

Reply:
Same as pytorch_trial, we already have `from determined import util`